app/src/main/java/org/dhis2/Bindings/Extensions.kt

```diff
@@ -67,3 +70,8 @@ fun Fragment.app(): App {
 fun AppCompatActivity.app(): App {
     return applicationContext as App
 }
+
+fun View.closeKeyboard(){
+    val imm = context.getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
+    imm.hideSoftInputFromWindow(windowToken, 0)
+}
```

Comment:
do we have a view extensions file? If not, I'll create one a move it there. "Extensions" might end up in a huge file